### ISSUE TITLE
synapse MERGE documentation doesn't call out limitation with identity column

### ISSUE THREAD

**Issue by jasonhorner** [MicrosoftDocs/sql-docs#5791]
in the merge document a note should be added saying that the merge statement doesn't support updating a table with a column that has the identity property set in synapse.  hopefully this would be fixed by GA.

https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/t-sql/statements/merge-transact-sql.md

more info here:
https://stackoverflow.com/questions/64563484/azure-synapse-merge-command-with-the-identity-column-in-target-table-is-not-wor

https://feedback.azure.com/forums/908035-sql-server/suggestions/41725273-azure-synapse-merge-command-with-the-identity-col

**Comment by WilliamAntonRohm** [comment]
@jasonhorner -- thank you for your feedback.

@XiaoyuMSFT -- please look into this issue.

**Comment by julieMSFT** [comment]
#assign:julieMSFT

**Comment by julieMSFT** [comment]
Hi @jasonhorner, thank you for contacting Microsoft. We have updated the documentation. Thank you for your contribution to Microsoft Docs. The document [MERGE (Transact-SQL)](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/t-sql/statements/merge-transact-sql.md) has been updated. I will go ahead with closing this issue. The @notification mechanism continues to work even after an item is closed, in case further communication is needed. We hope to hear from you again.
